gradle.properties

```diff
@@ -4,3 +4,5 @@ file.encoding=UTF-8
 kotlin.incremental=true
 org.gradle.daemon=true
 org.gradle.daemon.idleTimeout=3600000
+sonatypeUsername = System.getenv("SONATYPE_USERNAME")
+sonatypePassword = System.getenv("SONATYPE_PASSWORD")
```

Comment:
Putting it here forces everyone to use environment variables. And I do not want to put my password on an ENV on my dev machine. Because sometimes environment variables are dumped.

My proposal here is to try to select environments when available with fallback to properties.